src/ui/components/Widgets/Chart/Chart.tsx

```diff
@@ -170,7 +170,7 @@ export const Chart = (props: ChartNoWidgetProps) => {
         ignoreUsedParams,
         clearedOuterParams,
         onInnerParamsChanged,
-        enableActionParams: true,
+        enableActionParams: false,
```

Comment:
For this change we need to add new reserved get-parameter to enable it in preview. And add it;
- for forming link to open in new tab (and remove from link for edit, as far as I remember, there is single helper for both)
- support force adding get-parameter in mailings

Reply:
Added a separate parameter `_ap_enabled` (used for preview). The link generation for screenshots will be done in a separate task.